packages/autocomplete-js/src/autocomplete.ts

```diff
@@ -174,6 +174,7 @@ export function autocomplete<TItem extends BaseItem>(
       formElement: dom.value.form,
       panelElement: dom.value.panel,
       inputElement: dom.value.input,
+      isDetached: isDetached.value,
```

Comment:
`isDetached` is an Autocomplete JS API that we're leaking in Autocomplete Core now. It's not ideal.

I'm wondering if we could not attach `environmentProps.onTouchStart` to the element in Detached mode.

Reply:
You're right. We most surely can break the `touchstart` event by stopping it on the detached cancel button level.

Reply:
Done in c64a8611.